cmd/oras/internal/display/metadata/model/discover.go

```diff
@@ -18,16 +18,18 @@ package model
 import ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 
 type discover struct {
-	Manifests []Descriptor `json:"manifests"`
+	Subject   Descriptor   `json:"subject"`
+	Referrers []Descriptor `json:"referrers"`
 }
 
 // NewDiscover creates a new discover model.
-func NewDiscover(name string, descs []ocispec.Descriptor) discover {
+func NewDiscover(name string, subject ocispec.Descriptor, descs []ocispec.Descriptor) discover {
```

Comment:
nit: Maybe we can rename `descs` to `referrers` to make it clearer?

Reply:
done